doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -103,7 +103,9 @@ See `Zigbee samples`_ for the list of changes for the Zigbee samples.
 ESB
 ---
 
-|no_changes_yet_note|
+* Added:
+
+  * The :kconfig:option:`CONFIG_ESB_DYNAMIC_INTERRUPTS` Kconfig option to enable ESB to use direct dynamic interrupts
```

Comment:
```suggestion
  * The :kconfig:option:`CONFIG_ESB_DYNAMIC_INTERRUPTS` Kconfig option to enable direct dynamic interrupts.
```
Since we're in the ESB section, it's not necessary to specify it here.